dev/com.ibm.websphere.org.eclipse.microprofile.metrics.2.2/bnd.bnd

```diff
@@ -0,0 +1,42 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: com.ibm.websphere.org.eclipse.microprofile.metrics.2.2; singleton:=true
+
+Import-Package: \
+  javax.enterprise.util; version="[1.1,3)",\
+  *
+
+Export-Package: \
+  org.eclipse.microprofile.metrics;version=2.1, \
```

Comment:
I think this should be version 2.2. See https://github.com/eclipse/microprofile-metrics/blob/c1febc5c82fcd90c96aacbc0fa88917b1c16b286/api/src/main/java/org/eclipse/microprofile/metrics/package-info.java#L135